AIDevGallery/Controls/ModelSelectionControl.xaml.cs

```diff
@@ -265,9 +265,24 @@ private void PopulateModelDetailsLists()
             }
         }
 
+        SortModels(AvailableModels);
+        SortModels(DownloadableModels);
+        SortModels(UnavailableModels);
+
         SetHeaderVisibilityStates();
     }
 
+    private void SortModels<T>(ObservableCollection<T> models)
+        where T : IModelView
+    {
+        var sortedModels = models.OrderByDescending(m => m.ModelDetails.HardwareAccelerators.Contains(HardwareAccelerator.WCRAPI)).ToList();
+        models.Clear();
+        foreach (var model in sortedModels)
+        {
+            models.Add(model);
```

Comment:
Doing a clear and a bunch of adds will trigger multiple UI updates (INCC), which literally just happened on the method that calls this method.
It would be better to use regular lists on the PopulateModelDetailsLists method, and add only once to the OC.

Reply:
Removed sorting and just inserting apis on top now